packages/react/src/components/IonIcon.tsx

```diff
@@ -34,12 +34,15 @@ class IonIconContainer extends React.PureComponent<InternalProps> {
   }
 
   render() {
-    const { icon, ios, md, ...rest } = this.props;
+    const { icon, ios, md, mode, ...rest } = this.props;
 
     let iconToUse: typeof icon;
 
+    const config = getConfig();
```

Comment:
`getConfig` can return null so we should have a fallback for when that is the case: https://github.com/ionic-team/ionic-framework/blob/27a5356fa2b72073d565e9d6f527107869faa3ee/packages/react/src/components/utils/index.tsx#L31